utils/sites.js

```diff
@@ -344,7 +344,35 @@ const qc = (newData, existingData, rules) => {
     return errors;
 }
 
+const updateSignMechanism = async (req, res) => {
```

Comment:
Not to be super picky, but please update this.. I know what you're saying, but "sign mechanism" might be initially confusing to someone down the road. Suggestions at the top of my head would be something like "updateUserFirebaseAuthentication".

Reply:
I agree `updateSignMechanism` is not clear enough. I'd prefer `updateSignInMethod`, `updateUserAuthentication`, etc.